docs/0-what-is/0-introduction.mdx

```diff
@@ -12,264 +12,40 @@ slug: '/'
   src="/img/conduit/on-white-conduit-logo.png"
 />
 
-Conduit is a data integration tool for software engineers, powered by [Meroxa](https://meroxa.io). Its purpose is to
-help you move data from A to B. You can use Conduit to send data from Kafka to
-Postgres, between files and APIs,
-between [supported connectors](/docs/using/connectors/list),
+Conduit is a data streaming tool for software and data engineers, powered
+by [Meroxa](https://meroxa.io). Its purpose is to help you move data from A to
+B. You can use Conduit to send data from Kafka to Postgres, between files and
+APIs, between [supported connectors](/docs/using/connectors/list),
 and [any datastore you can build a plugin for](/docs/developing/connectors/).
 
-It's written in [Go](https://go.dev/), compiles to a binary, and is designed to
-be easy to use and [deploy](/docs/installing-and-running?option=binary).
+It's written in [Go](https://go.dev/), compiles to a single binary, and is
+designed to be easy to use and [deploy](/docs/installing-and-running).
 
 Out of the box, Conduit comes with:
 
-- Common connectors 
+- Built-in connectors 
 - Processors
 - Observability
 - Schema Support
 
-In this getting started guide we'll use a pre-built binary, but Conduit can also be run using [Docker](/docs/installing-and-running?option=docker).
+## Features
 
-## Some of its features
+Conduit comes with many powerful features such as:
 
-### It’s simple, yet powerful
+* **Simple deployments**: Conduit is distributed as a single binary and doesn't
+  require any external dependencies to run. We also provide a [Kubernetes
+  operator](/docs/scaling/conduit-operator).
+* **Real time by default**: Conduit pipelines continuously listen for changes to
+  a database, data warehouse, etc., and allows your data applications to act
+  upon those changes in real time.
+* **Easily extensible**: Conduit connectors are plugins that communicate with Conduit
+  via a gRPC interface. This means that plugins can be written in any language
+  as long as they conform to the required interface. Check out
+  our [connector docs](/docs/using/connectors/getting-started)!
 
-Eliminate the multi-step process you go through today. Just download the binary
-and start building. Conduit pipelines run in their own goroutines and are
-connected using Go channels. This makes Conduit pipelines incredibly performant
-on multi-core machines.
+The full list of features can be found on
```

Comment:
I don't think this is accurate, since these are _other_ features that aren't highlighted as first elements on the main navigation. I would probably rephrase this to something like

`For a detailed look at additional features, visit our [Features](/docs/using/other-features) page. Here, you'll find a selection of tools and functionalities that complement our main features.`

Reply:
Makes sense!